xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -504,9 +504,16 @@ bool CGUIControlFactory::GetActions(const TiXmlNode* pRootNode, const char* strT
     if (pElement->FirstChild())
     {
       CGUIAction::cond_action_pair pair;
+      std::string function;
+      vector<string> parameters;
+
       pair.condition = XMLUtils::GetAttribute(pElement, "condition");
       pair.action = pElement->FirstChild()->Value();
-      action.m_actions.push_back(pair);
+      // check if action is supported
+      CUtil::SplitExecFunction(pair.action, function, parameters);
+      if(actionsIgnore.size() == 0 || 
+          std::find(actionsIgnore.begin(), actionsIgnore.end(), function) == actionsIgnore.end())
```

Comment:
create a static map `TAGS_DISALLOWED_ACTIONS<tagname, vector<actions>>` and check the for the map key and and later for the action. everything should go to `CGUIControlFactory`